migrations/238_enum_role__drop.sql

```diff
@@ -0,0 +1,10 @@
+DELETE FROM assessment_access_rules WHERE role > 'Student';
+DELETE FROM course_instance_access_rules WHERE role > 'Student';
```

Comment:
We could also delete `enrollments WHERE role > 'Student'` as requested by @jonatanschroeder to ensure old course statistics are still correct.